src/test/ui/parser/macro/macro-doc-comments-1.stderr

```diff
@@ -5,7 +5,10 @@ LL | macro_rules! outer {
    | ------------------ when calling this macro
 ...
 LL |     //! Inner
-   |     ^^^^^^^^^ no rules expected this token in macro call
+   |     ^^^^^^^^^
+   |     |
+   |     no rules expected this token in macro call
+   |     inner doc comments expand to `#![doc = "..."]`, which is what this macro attempted to match
```

Comment:
"which is what this macro attempted to match" seems confusing. It seems to be using "macro" to mean "the compiler's expansion of this macro". But I would normally expect it to mean "the matchers defined in the macro", which makes this confusing - if it matches, why didn't it compile?

I think just the first half (this is a doc comment) is clear enough, maybe keep only that?

Maybe say something like

Reply:
@jyn514, I wanted to stress that the macro _sees_ the token stream `# ! [ doc =` and not `/// doc comment`, and that's why we get "unexpected `!` token". is there a clearer way to express this?

Reply:
I don't find this message confusing, so I'll approve and if we come up with something better then we change it to that in a follow up.